pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1666,6 +1671,10 @@ func (ctrl *PersistentVolumeController) findProvisionablePlugin(claim *v1.Persis
 			// External provisioner is requested, do not report error
 			return nil, class, nil
 		}
+		if ctrl.csiMigratedPluginManager.IsMigrationEnabledForPlugin(class.Provisioner) {
+			// CSI migration scenario with in-tree plugin disabled/removed
```

Comment:
shouldn't this find the migrated plugin even if In-tree plugin isn't disabled/removed?

Reply:
as in should we  check for migration before we even `FindProvisionablePluginByName`?